src/clr-angular/layout/tabs/tabs.ts

```diff
@@ -172,4 +164,29 @@ export class ClrTabs implements AfterContentInit, OnDestroy {
       sub.unsubscribe();
     });
   }
+
+  private listenForTabLinkChanges() {
+    return this.tabs.changes.pipe(startWith(this.tabs.map(tab => tab.tabLink))).subscribe(() => {
+      this._tabLinkDirectives = this.tabs.map(tab => tab.tabLink);
+      this.tabLinkElements = this._tabLinkDirectives.map(tab => tab.el.nativeElement);
+    });
+  }
+
+  private listenForOverflowMenuFocusChanges() {
+    return this.ifOpenService.openChange.pipe(filter(() => isPlatformBrowser(this.platformId))).subscribe(open => {
+      if (open && !this.inOverflow()) {
+        this.focusToFirstItemInOverflow();
+      } else if (!open && this.nextFocusedItemIsNotInOverflow()) {
+        this.keyFocus.resetTabFocus();
```

Comment:
We reset tab focus that way if you tab away and come back the focus isnt lost in a closed overflow